bots/images/scripts/lib/kubevirt.setup

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -ex
+
+KUBEVIRT_DIR="/kubevirt"
+rm -rf ${KUBEVIRT_DIR}
+mkdir -p ${KUBEVIRT_DIR}
+cd ${KUBEVIRT_DIR}
+
+# download latest Kubevirt release
+KUBEVIRT_RELEASE_ID=`curl -sL https://api.github.com/repos/kubevirt/kubevirt/releases/latest | grep -m 1 \"id\": | sed -e 's/^.*\"id\":\s*\(.*\),/\1/'`
```

Comment:
This calls the same github endpoint several times. We've had issues with getting ratelimited in the past when building images. Should we just call it once store it for reuse rather than calling it each time.

Reply:
done